docs/reference.rst

```diff
@@ -302,6 +302,23 @@ signature.
   can not be authenticated.
   If CRL checking is needed, the PKI needs to be structured with this in mind.
 
+``allowed-signer-cns=Name 1;Other Name`` (optional)
+  If this config parameter is set, RAUC will check the ``CommonName`` field
+  of the bundle's signer certificates against this semicolon-separated list.
+  If no signature from a certificate with a matching ``CommonName`` is found,
+  the target device is not eligible for the update bundle and the update process
+  is aborted.
+
+  This parameter can be used instead of checking the rauc system compatible
+  if only specific certificates in a shared PKI are trusted
+  for specific target devices.
+  The system compatible check could be overridden by a bundle
+  via the `install-check` hook.
```

Comment:
```suggestion
  This parameter can be used instead of relying on the RAUC system compatible check
  if only specific certificates in a shared PKI are trusted
  for specific target devices,
  as system compatible check could be overridden by a bundle
  via the `install-check` hook.
```